book/src/developer-guides/node-setup-guide/setup-docker.md

```diff
@@ -0,0 +1,33 @@
+## Setup docker and docker-compose
+
+```bash
+sudo apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common 
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt install docker-ce docker-ce-cli containerd.io
+```
```

Comment:
perhaps, use a [link](https://docs.docker.com/engine/install/) to official docs is better